src/helpers/getDeckAdvice/advice/INEFFICIENT_SATYR_COMBOS.js

```diff
@@ -0,0 +1,33 @@
+import toSentence from '../../toSentence/'
+import getRawCardData from '../../getRawCardData/'
+
+const getSatyrs = cards => cards.filter(c => c.race === 'satyr')
+const hasAny = (cards, ids) =>
+  ids.some(id => cards.map(card => card.id).includes(id))
+
+export default cards => {
+  const hasSatyrSynergist = hasAny(cards, ['S5', 'S7', 'S9'])
+  const satyrs = getSatyrs(cards)
+
+  if (!hasSatyrSynergist || satyrs.length > 5) return null
```

Comment:
Is 5 a fine number?

Reply:
I seldom use this faction, so I'm not sure. Might need input from a swarm veteran.